moderation-emails-override.php

```diff
@@ -0,0 +1,372 @@
+<?php
+
+if ( ! function_exists('wp_notify_postauthor') ) :
+/**
+ * Notify an author (and/or others) of a comment/trackback/pingback on a post.
+ *
+ * @since 1.0.0
+ *
+ * @param int|WP_Comment  $comment_id Comment ID or WP_Comment object.
+ * @param string          $deprecated Not used
+ * @return bool True on completion. False if no email addresses were specified.
+ */
+function wp_notify_postauthor( $comment_id, $deprecated = null ) {
+	if ( null !== $deprecated ) {
+		_deprecated_argument( __FUNCTION__, '3.8.0' );
+	}
+
+	$comment = get_comment( $comment_id );
+	if ( empty( $comment ) || empty( $comment->comment_post_ID ) )
+		return false;
+
+	$post    = get_post( $comment->comment_post_ID );
+	$author  = get_userdata( $post->post_author );
+
+	// Who to notify? By default, just the post author, but others can be added.
+	$emails = array();
+	if ( $author ) {
+		$emails[] = $author->user_email;
+	}
+
+	/**
+	 * Filters the list of email addresses to receive a comment notification.
+	 *
+	 * By default, only post authors are notified of comments. This filter allows
+	 * others to be added.
+	 *
+	 * @since 3.7.0
+	 *
+	 * @param array $emails     An array of email addresses to receive a comment notification.
+	 * @param int   $comment_id The comment ID.
+	 */
+	$emails = apply_filters( 'comment_notification_recipients', $emails, $comment->comment_ID );
```

Comment:
Since those are core hooks, you do not need to include the whole docblock. Something like this would do the trick:
`/** This filter is documented in core/src/wp-includes/pluggable.php */`